### ISSUE TITLE
Discord & Google Oauth

### ISSUE THREAD

**Issue by artrixdotdev** [benawad/dogehouse#340]
Since not everyone is a developer on DogeHouse, I think it might be a good idea to add discord and google oauth so people can sign up without github

**Comment by JammSpread** [comment]
Yeah, it may halt the money making of this billion dollar unicorn.

**Comment by yzx396** [comment]
@benawad I made Google Oauth working by leveraging github fields in users table (a bit hacky I know. But given I don't know how to add db fields in ECTO migration yet). I can share the code snippets if you like. Adding Google auth will be a boost of users as most people have google account...

**Comment by A-Yamout** [comment]
- [ ] Email Auth
- [ ] Google Auth
- [ ] 2FA

**Comment by lapwat** [comment]
+1 for email auth
+1 for no auth
